CommonData/dataset.h

```diff
@@ -109,7 +111,8 @@ class DataSet : public DataSetBaseNode
 	std::string					_dataFilePath,
 								_databaseJson;
 	
-	bool						_dataFileSynch			= false;
+	bool						_dataFileSynch			= false,
+								_showRSyntax			= false;
```

Comment:
Wouldn't you initialize this to `PreferencesModel::prefs()->showRSyntaxInResults()` somewhere?

As in, I dont see where it gets  initialized?